addons/gut/test.gd

```diff
@@ -1112,7 +1112,7 @@ func _assert_setget_called(type, name_property, setter = "", getter  = ""):
 	if name_getter != '':
 		expected_calls_getter = 1
 		stub(obj, name_getter).to_do_nothing()
-		var new_property = obj.get(name_property)
+		var _new_property = obj.get(name_property)
```

Comment:
To avoid any confusion between local variables/parameters that are ignored and class level variables, ignored variables and parameters should get two leading underscores.

Reply:
Sure, done.